presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1023,6 +1023,140 @@ Drop the schema ``iceberg.web``::
 
     DROP SCHEMA iceberg.web
 
+SHOW CREATE TABLE
+^^^^^^^^^^^^^^^^^
+
+Show the SQL statement that creates the specified Iceberg table by using ``SHOW CREATE TABLE``.
+
+For example, ``SHOW CREATE TABLE`` from the partitioned Iceberg table ``customer``:
+
+.. code-block:: sql
+
+    SHOW CREATE TABLE customer;
+
+.. code-block:: text
+
+    CREATE TABLE iceberg.tpch_iceberg.customer (
+        "custkey" bigint,
+        "name" varchar,
+        "address" varchar,
+        "nationkey" bigint,
+        "phone" varchar,
+        "acctbal" double,
+        "mktsegment" varchar,
+        "comment" varchar
+    )
+    WITH (
+        delete_mode = 'copy-on-write',
+        format = 'PARQUET',
+        format_version = '2',
+        location = 's3a://tpch-iceberg/customer',
+        partitioning = ARRAY['mktsegment']
+    )
+    (1 row)
+
+``SHOW CREATE TABLE`` from the un-partitioned Iceberg table ``region``:
+
+.. code-block:: sql
+
+    SHOW CREATE TABLE region;
+
+.. code-block:: text
+
+    CREATE TABLE iceberg.tpch_iceberg.region (
+        "regionkey" bigint,
+        "name" varchar,
+        "comment" varchar
+    )
+    WITH (
+        delete_mode = 'copy-on-write',
+        format = 'PARQUET',
+        format_version = '2',
+        location = 's3a://tpch-iceberg/region'
+    )
+    (1 row)
+
+SHOW COLUMNS
+^^^^^^^^^^^^^
```

Comment:
```suggestion
^^^^^^^^^^^^
```

Reply:
Yes, please! If you want to help make the overall doc more consistent and with fewer errors in the doc builds, I think it would be great.

Generally speaking, when I review with formatting, phrasing, or punctuation nits, I confine my edits to only the new content. I don't ask you to fix all the same nits throughout the doc file - especially if the file is large, or if the docs are part of a code-centric PR adding a new feature or fixing a code problem - because I don't want to slow down the velocity of the development. I'm okay with fixing the docs incrementally over time. Consistency in the docs is a long-term goal.